chapters/revisions.tex

```diff
@@ -1,9 +1,280 @@
 \chapter{Modelica Revision History}\label{modelica-revision-history}
 
-This appendix describes the history of the Modelica Language Design, and
-its contributors. This appendix is just present for historic reasons and
-is not normative. The current version of this document is available from
-\url{https://www.modelica.org/documents}.
+This appendix describes the history of the Modelica Language Design, and its contributors.
+This appendix is just present for historic reasons and is not normative.
+The current version of this document is available from \url{https://modelica.org/documents}.
+
+\section{Modelica 3.6}\label{modelica-3-6}
+
+Modelica~3.6 will later be voted to be approved.
+The Modelica~3.6 specification was edited by Hans Olsson.
```

Comment:
We used to have a separate section for each release listing all contributors. In 3.4 it was abbreviated to say "The members of the Modelica Association contributed to the Modelica 3.4 specification." And in 3.5 it disappeared completely. Was that intentional or accidental? I like the one sentence version and agree that we do not need a whole separate section for one sentence but wonder if some acknowledgement is still necessary. Or if that acknowledgement should not be in every release section but at the beginning of the chapter. After all, it is true for all versions.

Reply:
That was by accident. Will add it to maintenance, and for the next release add it together with the statement that a list of revisions will not be given in the future.